### ISSUE TITLE
Config object needs to be named "hparams"

### ISSUE THREAD

**Issue by andrewjong** [kourai-labs/mnist#7]
Epoch 1: 100%|██████████████████████████████████████████████████████████████████████████████| 938/938 [00:02<00:00, 355.76it/s, loss=0.495, v_num=0/home/andrew/.miniconda3/envs/pt-boiler/lib/python3.8/site-packages/pytorch_lightning/utilities/distributed.py:23: UserWarning: Did not find hyperparameters at model hparams. Saving checkpoint without hyperparameters.

I get this. Perhaps it needs to be explicitly named as "hparams"? Not sure.

**Comment by SBoulanger** [comment]
I ran train.py and did not get the same userwarning...

**Comment by SBoulanger** [comment]
Are you on the latest master ?
